src/platforms.md

```diff
@@ -1,41 +1,41 @@
 ---
 title: Platforms
-description: "You can use the Dart language to write mobile apps, web apps, standalone apps, servers, and more."
+description: "You can use the Dart SDK to write mobile apps, web apps, standalone apps, servers, and more."
 toc: true
 ---
 
-Using the Dart programming language and libraries, you can write simple scripts or
-full-featured apps. Whether you're creating a mobile app, web app, command-line
-script, or server-side app, there's a Dart solution for that.
+You can use Dart to write simple scripts or full-featured apps. Whether you're
```

Comment:
I like these words, but we're not supposed to use _Dart_ as a noun. :/

Reply:
Oh, same comment above. What can we use then that captures "all the things" we provide, as opposed to just the language?

Reply:
Resolving as per verbal discussion